modules/bitgo/test/v2/unit/coins/hbar.ts

```diff
@@ -49,6 +49,29 @@ describe('Hedera Hashgraph:', function () {
     explain.changeAmount.should.equal('0');
   });
 
+  it('should explain a token transfer transaction', async function () {
+    const tokenTransferParam = {
```

Comment:
seems a bit forcing, but that's the field needed in explainTransaction.
the former is using 1 line stringified json object which is not yet producable